### DISCUSSION TITLE
How to get annotation destinations (beyond pageNumber, coord location)?

### DISCUSSION THREAD

**Discussion by inf3rnus** [wojtekmaj/react-pdf#935] [Q&A]
Hey all!

Great freakin' package :)

So I understand that you can navigate to pages by using the page number via onItemClick().

However, I'm hoping to replicate the exact behavior of the pdf-js mozilla browser pdf reader, which follows anchor elements for each annotation.

Currently I don't see any way to get the annotation layer to properly render with the anchor's in the <a> tags...

Wondering if I'm just missing something silly, or if there's a magic combo of config options I need to use to get those to show up?

A page number and then a coord dest would be ideal, but yeah, any info would be greatly appreciated!

Thanks,
Aaron

**Comment by wojtekmaj** [comment]
Hmmm, I have no ready-to-go answer for you, but it seems like you could be interested in #812.
